use/faq.rst

```diff
@@ -105,11 +105,11 @@ I added an aka, but it doesn't work!
   instance, if you're adding an aka to give you a link to your
   homepage, you need to say::
 
-    aka add mylink "format concat http://my.host.com/ [urlquote $1]"
+    aka add mylink "format concat https://my.host.com/ [urlquote $1]"
 
   and not::
 
-    aka add mylink format concat http://my.host.com/ [urlquote $1]
+    aka add mylink format concat https://my.host.com/ [urlquote $1]
```

Comment:
I wonder if these should be using an example domain instead?

Reply:
yes they should